src/Badge.tsx

```diff
@@ -18,18 +19,22 @@ const propTypes = {
   bsPrefix: PropTypes.string,
 
   /**
-   * The visual style of the badge
-   *
-   * @type {('primary'|'secondary'|'success'|'danger'|'warning'|'info'|'light'|'dark')}
+   * A convenience prop for adding `bg-*` utility classes since they are so commonly used here.
+   * `light` and `dark` are common choices but any `bg-*` class is supported, including any custom ones you might define.
```

Comment:
this is still specific to the navbar calling out 'light' and 'dark', also still want the type into from the old comment as it it a bit more specific and still relevant

Reply:
Done. please check now.